pxc/jenkins/pxc56-pipeline.groovy

```diff
@@ -1,4 +1,6 @@
-pipeline_timeout = 10
+def pipeline_timeout = 10
```

Comment:
Just a question. I see that some of our groovy files have this `pipeline_timeout = 10` at the top, but then it is not used.
Does it make any sense? It should be then like 
```
timeout(time: pipeline_timeout, unit: 'HOURS')  {
   do something
}
```
I think it is some leftover that was copied over and over and it has no effect at all. Or do I miss something? @Percona-Lab/build-engineers

Reply:
@kamil-holubicki @VarunNagaraju 
Kamil is right here, this pipeline_timeout option looks like a leftover from some experiments and could be removed.